src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,25 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersAllValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
+            limit: Joi.string(),
```

Comment:
O campo limit também, poderia ter a validação de ser um número inteiro.

```suggestion
            limit: Joi
                .number()
                .integer()
                .positive()
                .max(50)
                .default(50),
```